obal/data/update_package.yml

```diff
@@ -5,12 +5,15 @@
   roles:
     - git_annex_setup
   tasks:
-    - name: 'Ensure upstream_files is defined'
+    - name: 'Fail if version not specified'
       fail:
-        msg: "'upstream_files' not defined for {{ inventory_hostname }}. Please define within package_manifest.yml"
-      when:
-        - upstream_files is not defined
-        - downstream_version is not defined
+        msg: "version must be specified"
+      when: version is not defined
+
+    - name: 'Create bit branch'
+      command: "git checkout -b {{ inventory_hostname }}-{{ version }}"
```

Comment:
For foreman-packaging I like to prepend ```rpm/``` or ```deb/```. Maybe a prefix parameter could be added?